casdoorsdk/util.go

```diff
@@ -16,12 +16,17 @@ package casdoorsdk
 
 import (
 	"bytes"
+	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"mime/multipart"
 	"strings"
 )
 
+// ErrCasdoorResponse returns while receiving invalid response from casdoor
+var ErrCasdoorResponse = errors.New("casdoor response")
```

Comment:
What's this? Is this better than the original error? I don't think so

Reply:
I was writting about it in issue: https://github.com/casdoor/casdoor-go-sdk/issues/60

Reply:
```
users, err := casdoorsdk.GetUsers()
if errors.Is(err, ErrCasdoorResponse) {
   // 
}```

Reply:
"casdoor response" this string is just not making sense.